src/chopper_layout.cpp

```diff
@@ -290,6 +290,20 @@ int execute(seqan3::argument_parser & parser)
 
     // brief Write the output to the result file.
     std::ofstream fout{config.output_filename};
+    // output configuration information
+    fout << "## Data input: " << config.data_file.string() << '\n'
+         << "## False positive rate: " << config.fp_rate << '\n'
+         << "## Number of hash fucntions: " << config.num_hash_functions << '\n'
+         << "## Tmax: " << config.t_max << '\n'
+         << "## Alpha: " << config.alpha << '\n';
+    if (!config.hll_dir.empty())
+    {
+        fout << "## HLL-input: " << config.hll_dir.string() << '\n'
+             << "## HLL-max-ratio: " << config.max_ratio << '\n'
+             << "## HLL-estimate-union: " << config.estimate_union << '\n'
+             << "## HLL-rearrange-bins: " << config.rearrange_bins << '\n';
+    }
+    // output layout header
```

Comment:
I like the idea, I am wondering whether:
* We should use YAML style (key:value is already YAML, keys also may have spaces). I think currently it's incidentally and not deliberately 😁 
* Use the long-ids as identifier
* Always print all options (all have a default, no?)
* Use another delimiter (like `@`). It doesn't really matter, but it would make parsing slightly easier since you just need to look at the first character of each line.

This way, we would have something like a config file; and we could actually read things like the FPR in raptor.

Reply:
Good points. I like the idea of it being similar then a config file. but then it might be worth it to take an afternoon and think about config files. Because if we want to output every option anyway, it would be nice if the argument parser can do this for us..